mods/persistence/modules/projectiles/overrides.dm

```diff
@@ -0,0 +1,38 @@
+/obj/item/gun/launcher/syringe/rapid
+	origin_tech = "{'wormholes':1337}"
+
+/obj/item/gun/energy/temperature
```

Comment:
I don't remember this gun being particularly good, and it doesn't have a new analogue. I don't think it needs to be made unobtainable. Ditto for a few others below

Reply:
as discussed on discord, will add variants for these guns in another pr soon